README.md

```diff
@@ -29,8 +29,8 @@ There are two main events to install the toolkit:
 2. Installing the Toolkit from the UI. This is performed by running the `InstallPythonToolkit` method. This will:
 	- Download Python and install python
 	- Download and install `pip`
-	- Install the necessary packages (e.g. `jupyter`)
-	- Install the python bindings of the Python_Engine from C:\ProgramData\BHoM\python-3.7.3-embed-amd64\src.
+	- Install the necessary packages (e.g. `jupyterlab`)
+	- Install the python bindings of the Python_Toolkit from C:\ProgramData\BHoM\python-3.7.3-embed-amd64\src.
```

Comment:
The Python Toolkit does not install to `C:\ProgramData\BHoM\python...` anymore - please double check the exact installation location and update this line. I think it's BHoM\Extensions but can't remember off the top of my head.